src/main/java/com/simleetag/homework/api/domain/home/api/dto/HomeWithMembersResponse.java

```diff
@@ -19,13 +19,13 @@ public record HomeWithMembersResponse(
 
         @Schema(description = "집에 속한 유저 목록")
         @NotBlank
-        List<MemberResponse> members
+        List<MemberWithUserResponse> members
 ) {
-    public static HomeWithMembersResponse from(Home home, List<MemberResponse> memberResponses) {
+    public static HomeWithMembersResponse from(Home home, List<MemberWithUserResponse> memberWithUserRespons) {
```

Comment:
typo가 있어용!
```suggestion
    public static HomeWithMembersResponse from(Home home, List<MemberWithUserResponse> memberWithUserResponses) {
```

밑 부분도 함께 변경되어야 할 것 같습니다